modules/gdnative/gdnative.cpp

```diff
@@ -48,7 +48,7 @@ static const bool default_reloadable = true;
 // Defined in gdnative_api_struct.gen.cpp
 extern const godot_gdnative_core_api_struct api_struct;
 
-Map<String, Vector<Ref<GDNative> > > *GDNativeLibrary::loaded_libraries = NULL;
```

Comment:
What was the rationale behind using a variable this way?
Just trying to learn :)

Reply:
I'm not sure exactly. Both usages use almost the same amount of memory, and while the previous approach defers the allocation to when it is actually needed, the size of an empty Map is pretty small.

I would guess that the writer of the initial code got confused by the way constructors of static variables are run, and decided to just write it with a pointer.

(CC @karroffel)

Reply:
I think I did it this way because I had problems with static initialization at the time, apparently I did something weird as it seems to work here properly.